### ISSUE TITLE
Impact of score_threshold on results

### ISSUE THREAD

**Issue by jdwinkler-lanzatech** [cmks/DAS_Tool#76]
Hi,

Thanks for your work on DAS_Tool, I have found it very useful in the past! I do have one question about the impact of the score threshold on the output. I noticed that DAS Tool is (correctly) discarding a lot of low quality bins for some of my samples. I generally like to do post-hoc assembly quality filtering with CheckM, so I was wondering if setting the score_threshold = 0 means that DAS Tool effectively just dereplicates the input MAGs only + discards any MAG with a negative score?

Thanks for your help!

**Comment by cmks** [comment]
Yes, that's correct. The output will be a de-replicated set of MAGs with score above 0 and MAGs with a score below 0 will not be reported. Following up with CheckM is a good idea.

**Comment by jdwinkler-lanzatech** [comment]
One additional question: DAS Tool is still removing bins that are presumably contaminated or megabins even with the score threshold of zero. I am guessing that behavior is probably what I want. How should I interpret the negative scores though?

**Comment by cmks** [comment]
Basically, a negative score means a high fraction of redundant single copy genes (SCGs). The score is calculated based on the number of unique- and redundant SCGs per bin. In case of a high number of redundant SCGs, the number of unique SCGs may get outweighed and the score can turn negative. You can always check the `SCG_completeness` and `SCG_redundancy` columns of the `.summary` file showing the unique- and redundant SCG fractions.
In case you want DAS Tool to report also bins with high SCG redundancy or even dereplicate your full bin set, you can do so by choosing a very low, negative score threshold.

**Comment by jolespin** [comment]
I have the same question.  My main goal is to dereplicate and not necessarily score the quality as I do that with CheckM post hoc.  Would you recommend setting the score to 0 in this case?

**Comment by 21f2000639** [comment]
After refining contig bins using the DAS tool, the summary.tsv file is generated, containing multiple columns with specific details. This issue aims to provide a brief explanation of each column to improve understanding of the refinement results. A clear interpretation of this data will assist in analyzing bin quality and making informed decisions for further analysis.
